core/src/main/kotlin/fr/sncf/osrd/graph/Pathfinding.kt

```diff
@@ -184,15 +186,23 @@ class Pathfinding<NodeT : Any, EdgeT : Any, OffsetType>(
             val startRange = EdgeRange(location.edge, location.offset, location.offset)
             registerStep(startRange, null, 0.0, 0, listOf(location))
         }
+        var maxReachedTarget = 0
         val start = Instant.now()
         while (true) {
             if (Duration.between(start, Instant.now()).toSeconds() >= timeout)
                 throw OSRDError(ErrorType.PathfindingTimeoutError)
-            val step = queue.poll() ?: return null
+            val step = queue.poll()
+            if (step == null) {
+                pathfindingLogger.info(
```

Comment:
Should we downgrade log level to debug for this (and maybe some others)?
I wonder if it'll spam logs (maybe up the aws costs a bit, but mostly make logs less readable).

Reply:
The logs in this PR can only display something once per request, which doesn't feel excessive. 

If we want to be more careful about what we log, maybe we should start with the warnings when loading an infra... Actually I'll open an issue for this